src/collector.rs

```diff
@@ -697,7 +697,9 @@ impl Recorder for RecorderHandle {
     fn record_histogram(&self, key: &Key, value: f64) {
         let _ = self.sender.try_send(Datum {
             key: key.clone(),
-            value: Value::Histogram(HistogramValue::new(value).unwrap()),
+            value: Value::Histogram(
+                HistogramValue::new(if value.is_finite() { value } else { 0.0 }).unwrap(),
```

Comment:
Given this infinity and negative infinity will be recorded as 0.0 as well as NaN, just wondering if this is preferred. Could non finite values be omitted instead?

Reply:
that was my thought as well.  
however, since we cant mess with the `metrics::histogram!()` macro, whatever we do is a choice. i wonder if we can have it as a setting in the builder somewhere.